CONTRIBUTING.md

```diff
@@ -6,6 +6,14 @@ The concord-bft project team welcomes contributions from the community. If you w
 signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
 questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq).
 
+## Important
+
+Concord-bft is used in multiple commercial projects which are naturally prioritized by the concord team.
+
+Sometimes there are delays until code and documentation is merged into this repository, mainly because of the need to strip proprietary information from the customers systems.
+
+There is no separate commercial version - everything related to concord-bft is mergend back into this repo.
```

Comment:
> "There is no separate commercial version - everything related to concord-bft is mergend back into this repo."

Well, actually concord-bft is a library used "as-is" by the actual VMware Blockchain product. The changes are first made in concord-bft and then "flow" into the product which adds support for various execution engines (EVM, DAML, Hyperledger chaincode), automation, fleet management, monitoring, etc.

Reply:
This is based on the comments re "paying customers" within https://github.com/vmware/concord-bft/issues/426#issuecomment-594690171. This is to clarify

a) some docs have delays in getting into the repo (as mentioned within https://github.com/vmware/concord-bft/issues/426#issuecomment-594690171 - i assumed that the same is true for some code).

## Questions:
*  you have support for DAML now already? 
* with "product" you mean https://github.com/vmware/concord?

Reply:
Hi @lazaridiscom , VMware Blockchain product provides the option to run either EVM, DAML, or Hyperledger as the execution engine. We have only open-sourced the integration wth EVM for now, which is the GitHub repo you refer to:  https://github.com/vmware/concord

Reply:
for DAML: https://github.com/vmware/concord/issues/33

"Hyperledger as the execution engine" can you please clariy, as "Hyperledger" has a many projects with different languages / engines. To which "execution engine" of Hyperledger you refer?

(I have now understood better, my misunderstanding was that you open-source all code, but https://github.com/vmware/concord-bft/issues/426#issuecomment-594690171 refers to the BFT code.

Possibly it is the best to remove this part completely for now.)

Reply:
As you know, Concord BFT is a generic state machine replication library. Any state machine execution engine out there can be plugged into the consensus algorithm. One of the supported engines is the Hyperledger Chaincode execution engine.

As for DAML, we haven't yet considered open sourcing this part of the product. The enterprise blockchain field is an emerging one, so we are still working on identifying the right proportion of open source vs licensed/SaaS.

In any case, we consider our biggest contribution to the community to be the Concord BFT library. Given this library, one can build any blockchain they wish, or any BFT distributed system imaginable.

Reply:
By the way, we have a very strong system testing suite that publicly proves the safety properties of the Concord-BFT consensus algorithm, using a "Simple K/V blockchain" application. You can have a look here:
https://github.com/vmware/concord-bft/tree/master/tests/apollo

Reply:
> Hyperledger Chaincode execution engine

The one from within HL Fabric (go and Java, to my knowledge)?